carma_wm_ctrl/config/parameters.yaml

```diff
@@ -1,13 +1,20 @@
 #Integer; The number of times it publishes Geofence Acknowledgement.
 ack_pub_times: 1 
+
 #Double; Max lane width in meters within which geofence points are associated to a lanelet as those points are guaranteed to apply to a single lane
 max_lane_width: 4.0
 
+#Double: How much more in meters should the TCR bounding box be expanded
+tcr_bbox_expansion_meters: 50.0
+
 #Double: Period in seconds between traffic control requests after route selection
+#        This helps avoid certain situations where the box's angle
```

Comment:
These two lines need to be moved up to the `tcr_bbox_expansion_meters` parameter